src/python/tests_extended/test_dft_based.py

```diff
@@ -0,0 +1,311 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------------------------
+
+import os
+import sys
+import io
+import platform
+import tempfile
+import unittest
+
+import numpy as np
+import pandas as pd
+from nimbusml import Pipeline
+from nimbusml.preprocessing.schema import ColumnSelector
+from nimbusml.preprocessing import ToString, ToKeyImputer, DateTimeSplitter
+from scipy.sparse import csr_matrix
+from nimbusml.timeseries import TimeSeriesImputer, LagLeadOperator, RollingWindow, ForecastingPivot, ShortDrop
+from nimbusml.preprocessing import (TensorFlowScorer, FromKey, ToKey,
+                                    DateTimeSplitter, OnnxRunner)
+import onnxruntime as rt
+from data_frame_tool import DataFrameTool as DFT
+
+TEST_CASES = {
+    'DateTimeSplitter_Simple',
+    'DateTimeSplitter_Complex',
+    'ToKey_SimpleFloat',
+    'ToKey_SimpleDouble',
+    'ToKey_SimpleString',
+    'ToKey_2col_Double',
+    'ToKey_2col_Double_String',
+    'ToString_Numbers',
+    'ToString_Other_Types',
+    'TimeSeriesImputer_1grain_2gap',
+    'TimeSeriesImputer_1grain_1gap_2filtercolumn',
+    'TimeSeriesImputer_2grain_nogap',
+    'ShortGrainDropper',
+    'RollingWin_Pivot_Integration',
+    'Laglead_Pivot_Integration',
+}
+
+INSTANCES = {
+    'DateTimeSplitter_Simple': Pipeline([
+        DateTimeSplitter(prefix='dt') << 'tokens1',
+        ColumnSelector(drop_columns=[
+                                    'dtHour12', 'dtDayOfWeek', 'dtDayOfQuarter',
+                                    'dtDayOfYear', 'dtWeekOfMonth', 'dtQuarterOfYear',
+                                    'dtHalfOfYear', 'dtWeekIso', 'dtYearIso', 'dtMonthLabel',
+                                    'dtAmPmLabel', 'dtDayOfWeekLabel', 'dtIsPaidTimeOff','dtHolidayName'
```

Comment:
Add few more tests where you dont drop the columns and do compare them

Reply:
Further tests need to be added after support for countries is done.